packages/indicators/src/types.ts

```diff
@@ -87,3 +87,5 @@ export type FetchOptions = Readonly<{
   readonly endDate: string;
   readonly period?: string;
 }>;
+
+export type DataBroker = unknown;
```

Comment:
This type is enough here since `DataBroker` is just passed through from `indicators` to `aggregator`.

A more specific type can be imported from `aggregator` if that package gets migrated to TS